lib/media/drm_engine.js

```diff
@@ -1073,12 +1089,15 @@ shaka.media.DrmEngine.prototype.sendLicenseRequest_ = function(event) {
     this.unpackPlayReadyRequest_(request);
   }
 
-  this.networkingEngine_.request(requestType, request)
+  this.playerInterface_.netEngine.request(requestType, request)
       .then(function(response) {
         if (this.destroyed_) return Promise.reject();
 
         // Request succeeded, now pass the response to the CDM.
         return session.update(response.data).then(function() {
+          var event = new shaka.util.FakeEvent('drmsessionupdate');
```

Comment:
Please document this event type in lib/player.js.  You'll see several examples toward the top of the file.

Reply:
Done.